stylelint-polaris/index.js

```diff
@@ -304,9 +304,10 @@ const stylelintPolarisCoverageOptions = {
         /\$border-width-data/,
         /\$borders-data/,
         // Legacy custom properties
-        // /--p-border-radius-base/,
+        /--p-border-radius-base/,
+        /--p-border-radius-large/,
+        /--p-border-radius-half/,
```

Comment:
Yes. Removing these tokens is a bit weird to add them to the linter. The reason I think we should do this is to encourage people to use the latest version and use the tokens as intended. At some point in the future this can definitely be removed.